Cargo.lock

```diff
@@ -3181,6 +3562,34 @@ dependencies = [
  "try-lock 0.2.3",
 ]
 
+[[package]]
```

Comment:
Curious, with all the new dependencies, how much did we regress on footprint?

Reply:
So sorry! I thought I replied to this:
Without Prometheus:
- agent: 24,448,848 
- controller: 17,427,432
- udev-broker: 9,873,408 
Each of the components increase by the following amount with Prometheus support added:
agent : 2,437,008
controller: 3,107,632
udev-video-broker: 2,525,352